### ISSUE TITLE
CSP issue when Nextcloud is in a subdir

### ISSUE THREAD

**Issue by jeromelebleu** [nextcloud/richdocuments#2050]
**Describe the bug**
When Nextcloud is in a subdir, Collabora doesn't load. We stay on a blank page in Nextcloud 22 and on *Document loading failed* in Nextcloud 23.

This issue is the same than this [one in help.nextcloud.com](https://help.nextcloud.com/t/refused-to-send-form-data-to-because-it-violates-the-following-content-security-policy-directive-form-action-self/133767).

**To Reproduce**
1. Install Nextcloud in a subdir - e.g. `/nextcloud/`
2. Open a file with Collabora from the Files app in your Web browser

## Server details

**Nextcloud version:** 22, 23

**Version of the richdocuments app:** 4.2.4, 5.0.2

**Version of Collabora Online:** 21.11.1.4-1

<details>
<summary>Logs</summary>

#### Browser log

The JavaScript console log in Chromium:
```
jquery.js:510 Refused to send form data to 'https://collbora.example.org/browser/739da71/cool.html?WOPISrc=https%3A%2F%2F…7mn&title=File.odt&closebutton=1&revisionhistory=1' because it violates the following Content Security Policy directive: "form-action 'self'".
```
</details>

**Comment by jeromelebleu** [comment]
This is introduced by https://github.com/nextcloud/richdocuments/pull/1977, [this check](https://github.com/nextcloud/richdocuments/pull/1977/files#diff-4eb824426a8cffb8b33b62b57744c69f32ea30ea755249a7a6170d9402885debR203) always returns since it is in a subdir. When this line is removed, Collabora loads normally.

**Comment by juliushaertl** [comment]
Fixed with #2051 and released as 5.0.3 as mentioned above.
